### PR TITLE
Add button for multiple plans

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds ability to use combined plans.

### CODE REVIEW

File: static/js/ducks/gcnEvent.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ export function deleteCommentOnGcnEvent(gcnEventID, commentID) {
 
 export const submitObservationPlanRequest = (params) => {
   const { instrument_name, ...paramsToSubmit } = params;
+  console.log("paramsToSubmit", paramsToSubmit);
```
Comment (Theodlz) [inline static/js/ducks/gcnEvent.js:91]: maybe removing that unless it's for debug

**Review by Theodlz — COMMENTED** [review]
Hi Michael, i'm not familiar with the obs plan to know what this should do / why this is needed. Could you explain it to me a bit more ?

**Comment by mcoughlin** [comment]
@Theodlz So we setup the interface so we can submit multiple plans at once, but they were still run in serial (i.e. the plans were separate). Now, if you check this button, they both go to gwemopt such that the schedules "know" about each other to credit a coordinated plan.

**Comment by Theodlz** [comment]
Ok ! Sounds great. I'll review the code later today then 👍

**Review by guynir42 — APPROVED** [review]
LGTM

File: skyportal/facility_apis/observation_plan.py [hunk 2]
```diff
@@ -21,7 +21,264 @@
...
+        Galaxy,
+        InstrumentField,
+        ObservationPlanRequest,
+        PlannedObservation,
+        User,
+    )
+
+    plans, requests = [], []
+
+    session = Session()
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply skyportal/facility_apis/observation_plan.py:48]: Just because the scoped sessions have behaved weirdly using the with syntax.

**Pull request merged**